### PR TITLE
fix HP bug with field effect strength and mark japanese as supported

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: pbrEngine/memorymap/values.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def getLanguage(text):
 
 
 WII_LANGUAGES = [
-    Language("Japanese", "ja", 0, False),  # Not PAL, so incompatible with pbrEngine
+    Language("Japanese", "ja", 0, True),  # PAL language replacement has been provided
```
Comment (deadinsky) [inline pbrEngine/memorymap/values.py]: weeb

File: pbrEngine/engine.py [hunk 2]
```diff
@@ -1331,6 +1331,8 @@ def _quitMatch(self):
         logger.debug("Quitting match")
         self._resetBlur()
         self.setVolume(0)  # Mute match setup beeping
+        self._matchFieldEffectStrength = 1.0
+        self._setFieldEffectStrength(self._matchFieldEffectStrength)
```
Comment (Felk) [inline pbrEngine/engine.py]: what does it do?

Comment (red031000) [reply pbrEngine/engine.py]: if the field effect strength is set to 0 the match prior, the next match will not have HP bars, by setting the field effect strength back to 1 when the match ends, it should fix that issue

Comment (Felk) [reply pbrEngine/engine.py]: I feel like things like this should be done _before_ each match, not after

**Comment by MathyFurret** [comment]
Make sure to reset the FoV as well, as that can also cause the issue

File: pbrEngine/engine.py (hunk 2 above)
Comment (red031000) [reply pbrEngine/engine.py]: it is done before each match at the current moment, which causes the HP bar issue

Comment (red031000) [reply pbrEngine/engine.py]: to clarify, the field effect strength is set in _matchStartDelayed, the fov is set in a gevent spawned in _matchStart

**Comment by red031000** [comment]
fov can't be set to 0, so I'm not sure if it'll cause the issue

**Pull request merged**